exporter/prometheusexporter/collector.go

```diff
@@ -403,4 +428,50 @@ func (c *collector) Collect(ch chan<- prometheus.Metric) {
 		ch <- m
 		c.logger.Debug(fmt.Sprintf("metric served: %s", m.Desc().String()))
```

Comment:
Doesn't need to be in this PR but as a followup would be good to clean up this sprintf as well